### PR TITLE
Added the option retainParsedDXF and a GetDXF accessor

### PR DESCRIPTION
**Pull request by jdh-invicara**
Retaining the result from DxfParser to allow for further processing (e.g. file conversion, partial-geometry extraction).  

Because this will use a lot of memory it is optional with the flag called retainParsedDXF with a default value of `false`

Added GetDXF() to DxfViewer to return the object to the client.

### CODE REVIEW

File: package-lock.json
Comment (vagran) [inline package-lock.json:1]: Do not commit this file since it is a library which is a dependency itself.

File: src/DxfViewer.js [hunk 1]
```diff
@@ -115,6 +115,10 @@ export class DxfViewer {
         return this.canvas
     }
 
+    GetDXF() {
```
Comment (vagran) [inline src/DxfViewer.js:118]: Please follow naming convention used in the project, capitalise abbreviations like `GetDxf()`. The same for the rest new symbols, like `parsedDxf`, `retainParsedDxf`.

File: src/DxfWorker.js [hunk 2]
```diff
@@ -155,9 +155,10 @@ export class DxfWorker {
         if (progressCbk) {
             progressCbk("prepare", 0, null)
         }
+        // console.log(">>>", JSON.stringify(dxf, null, 2))
```
Comment (vagran) [inline src/DxfWorker.js:158]: Please remove this commented code.

**Review by vagran — CHANGES_REQUESTED** [review]

File: package-lock.json
Comment (jdh-invicara) [reply package-lock.json:1]: Done

**Pull request merged**